src/nedap/utils/test/impl.cljc

```diff
@@ -40,3 +41,22 @@
        (if (cljs.test/successful? summary)
          (set! (.-exitCode js/process) 0)
          (set! (.-exitCode js/process) 1)))))
+
+(spec/def ::to   some?)
+(spec/def ::from some?)
+(spec/def ::to-change some?)
+(spec/def ::expect-options
+  (spec/keys :req-un [::to-change
+                      ::from
+                      ::to]))
+
+(defn expect
+  [bodies {:keys [to-change from to] :as options} clj?]
+  {:pre [(spec/valid? (complement empty?) bodies)
+         (spec/valid? ::expect-options options)
+         (spec/valid? boolean? clj?)]}
+  (let [is (if clj? 'clojure.test/is 'cljs.test/is)]
+    `(do
+       (~is (= ~to-change ~from) "Initial state doesn't match!")
```

Comment:
the first `is` should be an `assert`ion instead,  that way reporters don't bloat their assertion count.

See also https://github.com/nedap/clojure-guidelines/blob/339b8f0004a6d2f99ee1d8cb01ca2604074b83b0/testing.md#occasional-use-assert-to-denote-preconditions

Reply:
Interesting suggestion! Sure makes usage clearer. Will fix